dom.bs

```diff
@@ -2195,6 +2195,10 @@ steps:</p>
  <li><p>If the <i>open flag</i> is set and <var>shadow</var>'s <a for=ShadowRoot>mode</a> is
  <em>not</em> "<code>open</code>", then return null.</p></li>
 
+ <li><p>If <var>shadow</var>'s <a for=ShadowRoot>slot assignment</a> is set to "<code>manual</code>",
+ return the <a>slot</a> in <var>shadow</var>'s <a for=tree>descendants</a> whose <a>manually assigned nodes</a>
+ includes <var>slottable</var>, if any, and null otherwise.</p></li>
```

Comment:
you want `<a for=set>contains</a>` instead of includes here I think. It's a set right?

And I guess we're not bothering with tree order and such since there can only be one such slot.

Reply:
Thanks.  Yes, contains is correct. I see how the verbiage is being used. 
Right, there's can only be one slot because the corresponding HTML spec for HTMLSlotElement::assign() ensures that.